components/scream/src/share/tests/common_physics_functions_tests.cpp

```diff
@@ -274,6 +290,13 @@ void run(std::mt19937_64& engine)
     // Compute vmr from mmr and vice versa
     PF::calculate_vmr_from_mmr(team,h2o_mol,qv,mmr_for_testing,vmr);
     PF::calculate_mmr_from_vmr(team,h2o_mol,qv,vmr,mmr);
+
+    // Compute wetmmr assuming mmr is dry
+    PF::calculate_wetmmr_from_drymmr(team,mmr,qv,wetmmr);//get wetmmr from mmr, assuming mmr is drymmr
+
+    // Convert wetmmr computed above to drymmr (i.e. mmr)
+    PF::calculate_drymmr_from_wetmmr(team,wetmmr,qv,mmr);//convert it back to drymmr, i.e. mmr
```

Comment:
The variable `mmr` is already being used for the vmr/mmr tests and is still being used later in the test code, see line 342.  Could we create a new variable, call it `mmrdry` that you could use for the wet/dry tests so that these don't conflict?

Reply:
Thanks! It is a good idea. I created a new variable named `drymmr0`. My tests were converting mmr back to its original value, so I didn't create a new variable earlier. You are right, it is not safe to rely on that.